lib/utils.pm

```diff
@@ -1077,4 +1078,20 @@ sub script_run_interactive {
     }
 }
 
+
+=head2 sed_config
+  sed_config("filename",{text_to_find => text_to_replace})
+
+  generify sed usage as config file modification tool.
+  allow to modify several items in one function call 
+  by providing  text_to_find / text_to_replace as hash key/value pairs
+=cut
+sub sed_config {
+	my($filename, $to_replace) = @_;
+	foreach my $key (keys %{$to_replace}) {
+		assert_script_run(sprintf("sed 's/%s/%s/' -i %s",$key,$to_replace->{$key},$filename));
```

Comment:
Just in case, we should escape `'` with `'"'"'` for key and value.

Reply:
can you post a suggestion here , not sure how to do it ...

Reply:
```
  foreach my $key (keys %to_replace) {
        my $value = $to_replace{$key};
        $value =~ s/'/'"'"'/g;
        $key =~ s/'/'"'"'/g;
        assert_script_run(sprintf("sed -E 's/%s/%s/%s' -i %s", $key, $value, $sed_modifier, $filename));
    }
```